test/unit/inspec_tools/xccdf_test.rb

```diff
@@ -42,8 +42,17 @@ def test_to_inspec_metadata
   end
 
   def test_controls_count
-    xccdf = InspecTools::XCCDF.new(File.read('examples/xccdf2inspec/data/U_Red_Hat_Enterprise_Linux_7_STIG_V1R4_Manual-xccdf.xml'))
+    xccdf = InspecTools::XCCDF.new(File.read('examples/xccdf2inspec/data/U_RHEL_7_STIG_V3R3_Manual-xccdf.xml'), true)
     inspec_json = xccdf.to_inspec
-    assert_equal(240, inspec_json['controls'].count)
+    assert_equal(249, inspec_json['controls'].count)
+  end
+
+  def test_control_id_flag
+    xccdf = InspecTools::XCCDF.new(File.read('examples/xccdf2inspec/data/U_RHEL_7_STIG_V3R3_Manual-xccdf.xml'), true)
+    vulnID_inspec_json = xccdf.to_inspec
+    xccdf = InspecTools::XCCDF.new(File.read('examples/xccdf2inspec/data/U_RHEL_7_STIG_V3R3_Manual-xccdf.xml'), false)
+    ruleID_inspec_json = xccdf.to_inspec
+    assert(vulnID_inspec_json['controls'].first['id'].start_with? 'V-')
+    assert(ruleID_inspec_json['controls'].first['id'].start_with? 'SV-')
```

Comment:
To make it easier to understand this test I'd say this should probably be broken out into 2 different tests, 1 for the true and one for the vulnID and one for the ruleID.

Reply:
Changes made